LEA_Symbol_Library/LEA_Symbol_Library.kicad_sym

```diff
@@ -25487,6 +25756,59 @@
       (effects (font (size 1.27 1.27)) hide)
     )
   )
+  (symbol "R_0402_60R4_THICK_0.5P_50V" (pin_numbers hide) (pin_names (offset 0.254) hide) (in_bom yes) (on_board yes)
```

Comment:
Same here: What is the purpose for the very small 0402 component? Why not using 0603?